Server/ConcordiaCurriculumManager/Security/AuthorizationPolicies.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Authorization;
+
+namespace ConcordiaCurriculumManager.Security;
+
+public static class AuthorizationPolicies
```

Comment:
Looks good, I'm just wondering what's AuthorizationPolicies for?

Reply:
This part is to add custom authorization policies which can then be used as part of the `[Authorize]` attribute. For example, one can create a policy to allow only admins to access certain endpoints.

For example,
`options.AddPolicy("AdminOnly", policy => policy.RequireRole("Admin"));` 
Which can then be used as 
`[Authorize(Policy = "AdminOnly")]`

Policies can be requirements as well and not just simple roles.